components/plate_based_widgets/stimulationstudio/StimulationStudioWidget.vue

```diff
@@ -1,59 +1,81 @@
 <template>
   <div>
-    <!-- original mockflow ID: id="cmpD1d83165d7321a67ce408a4ab3362bc9e" -->
     <div class="div__simulationstudio-backdrop"></div>
-    <!-- original mockflow ID: id="cmpDab1a64dec495767a5b74b08ff7a4e5b9" -->
-    <div class="div__simulationstudio-plus-icon-container">
-      <!-- original mockflow ID: id="cmpDab1a64dec495767a5b74b08ff7a4e5b9_txt" -->
-      <span class="span__simulationstudio-plus-icon">
-        <FontAwesomeIcon :icon="['fa', 'plus-circle']" />
-      </span>
+    <span
+      v-for="column_index in 6"
+      :key="'column_' + column_index"
+      :style="'left:' + column_left_offset(column_index) + 'px;'"
+      class="span__stimulationstudio-column-index"
+    >
+      <label
+        :id="'column_' + column_index"
+        @click.exact="on_select(column_index, 'column')"
+        @click.shift.exact="on_shift_click(column_index, 'column')"
+        @mouseenter="on_enter_hover(column_index, 'column')"
+        @mouseleave="on_leave_hover(column_index, 'column')"
+        >0{{ column_index }}</label
+      >
+    </span>
+    <span
+      v-for="(value, key) in row_values"
+      :key="'row_' + key"
+      :style="'top:' + row_top_offset(key) + 'px;'"
+      class="span__stimulationstudio-row-index"
+    >
+      <label
+        :id="'row_' + key"
+        @click.exact="on_select(key, 'row')"
+        @click.shift.exact="on_shift_click(key, 'row')"
+        @mouseenter="on_enter_hover(key, 'row')"
+        @mouseleave="on_leave_hover(key, 'row')"
+      >
+        {{ key }}</label
+      >
+    </span>
+    <span
+      class="span__stimulationstudio-toggle-plus-minus-icon"
+      @click.exact="on_select_cancel_all(all_select_or_cancel)"
+      @mouseenter="on_plus_minus_enter_hover(all_select_or_cancel)"
+      @mouseleave="on_plus_minus_leave_hover(all_select_or_cancel)"
+    >
+      <FontAwesomeIcon v-show="all_select_or_cancel" id="plus" :icon="['fa', 'plus-circle']" />
+      <FontAwesomeIcon v-show="!all_select_or_cancel" id="minus" :icon="['fa', 'minus-circle']" />
+    </span>
+    <div v-for="well_index in Array(24).keys()" :key="well_index">
```

Comment:
There's talk about eventually supporting a 96 well plate so it might be good to parameterize the number of wells

Reply:
Got it!